### PR TITLE
Convert to using Bootstrap

### PR DESCRIPTION
**Pull request by danieldortega77**
Make the webpage have a navigation bar, more images, and overall better appearance.

### CODE REVIEW

**Review by dslucas — APPROVED** [review]
HTML looks clean! Let's wait for @dborkan to take a look before merging.

File: portfolio/pom.xml [hunk 1]
```diff
@@ -35,7 +35,7 @@
         <version>2.2.0</version>
         <configuration>
           <!-- TODO: set project ID. -->
```
Comment (dborkan) [inline portfolio/pom.xml]: NIT: you can remove this TODO now

**Review by dborkan — APPROVED** [review]
Just one tiny nit, otherwise looks good!

File: portfolio/pom.xml (hunk 1 above)
Comment (danieldortega77) [reply portfolio/pom.xml]: Done!

**Pull request merged**